src/app/products/charges/edit-charge/edit-charge.component.ts

```
@@ -92,8 +94,15 @@ export class EditChargeComponent implements OnInit {
       ],
       active: [this.chargeData.active],
       penalty: [this.chargeData.penalty],
-      minCap: [this.chargeData.minCap],
-      maxCap: [this.chargeData.maxCap],
+      minCap: [
+        this.chargeData.minCap || null,
+        ,
+        [maxNumberValueValidator('maxCap')]
+      ],
```

Biome (2.1.2)

[error] This array contains an empty slots..

The presences of empty slots may cause incorrect information and might be a typo.
Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)